qiskit/transpiler/passes/basis/unroller.py

```diff
@@ -70,6 +70,9 @@ def run(self, dag):
 
             # TODO: allow choosing other possible decompositions
             try:
+                phase = 0
+                if node.op.definition and node.op.definition.global_phase:
```

Comment:
This `if` statement might not be necessary since it's caught by the `try` statement as would occur in the `while` loop below.